packages/react/src/hooks/useManageSubscription.ts

```diff
@@ -103,7 +101,6 @@ export const useManageSubscription = (
   }, [web3inboxClientData, account, domain]);
 
   if (!web3inboxClientData) {
-    console.log(">>> useManageData no client data");
```

Comment:
Let's remove the unnecessary console logs. We should still use them while doing debugs but we shouldn't include them on the production.